src/plotman/resources/plotman.yaml

```diff
@@ -59,31 +62,68 @@ directories:
                 - /mnt/dst/01
 
         # Archival configuration.  Optional; if you do not wish to run the
-        # archiving operation, comment this section out.
+        # archiving operation, comment this section out.  Almost everyone
+        # should be using the archival feature.  It is meant to distribute
+        # plots among multiple disks filling them all.  This can be done both
+        # to local and to remote disks.
+        #
+        # As of v0.4, archiving commands are highly configurable.  The basic
+        # configuration consists of a script for checking available disk space
+        # and another for actually transferring plots.  Each can be specified
+        # as either a path to an existing script or inline script contents.
+        # It is expected that most people will use existing recipes and will
+        # adjust them by specifying environment variables that will set their
+        # system specific values.  These can be provided to the scripts via
+        # the `env` key.  plotman will additionally provide `source` and
+        # `destination` environment variables to the transfer script so it
+        # knows the specifically selected items to process.  plotman also needs
+        # to be able to generally detect if a transfer process is already
+        # running.  To be able to identify externally launched transfers, the
+        # process name and an argument prefix to match must be provided.  Note
+        # that variable substitution of environment variables including those
+        # specified in the env key can be used in both process name and process
+        # argument prefix elements but that they use the python substitution
+        # format.
```

Comment:
I expanded a little bit the comments. PS: I think it's better to write it on the Wiki and add the link here. 

```yaml
# Archival configuration, used to move finished plots between
# directories or servers if you don't want to have them on the
# default local chia writes them. 
# 
# Whilst configuring archival is option, it's highly encouraged
# to do so, especially if you have more complex environments. It is 
# meant to distribute plots among multiple disks making the most
# usage of available space. 
# 
# If you don't plan to use archival, just comment this entire section. 
# 
# As of v0.4, archiving commands are highly configurable. The basic
# configuration consists of a script that checks available disk space
# and another for actually transferring the plots. Each can be specified
# as either a path to an existing script or inline script contents.
# 
# There's no technical difference between specifying an external
# script or adding it inline, it all comes down to your own preference. 
# 
# The default recipes provided by plotman should cover most common 
# scenarios, therefore you don't need to set anything else besides
# enabling it and choosing between local or remote and adjusting
# some of the default values. 
# 
# The `env` key it's a general purpose section to add any custom
# variable and value you want to make available to the underlying
# script. See the provided examples. 
# 
# On top of the keys and values that you set up in `env`, plotman provides 
# the variables `source` and `destination` to the transfer script 
# so it knows the specifically selected items to process.  
# 
# Also, plotman needs to be able to detect if a transfer process is already
# running. To be able to identify externally launched transfers, the
# process name and an argument prefix to match must be provided.  Note
# that variable substitution of environment variables including those
# specified in the env key can be used in both process name and process
# argument prefix elements but that they use the python substitution
# format.
```